.travis.yml

```diff
@@ -7,9 +7,13 @@ git:
 before_install:
   - sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
   - sudo apt-get update -qq
-  - sudo apt-get install -y python-ply python-git libelf-dev codespell sparse fakeroot gcc-7 g++-7 gcc-aarch64-linux-gnu
-  - sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-7 70 --slave /usr/bin/g++ g++ /usr/bin/g++-7
+  - sudo apt-get install -y python-ply python-git libelf-dev codespell sparse fakeroot gcc-9 g++-9 gcc-aarch64-linux-gnu
+  - sudo apt-get install -y xz-utils lftp
```

Comment:
There seems to be a fancier way now: https://docs.travis-ci.com/user/installing-dependencies#installing-packages-with-the-apt-addon
```
addons:
  apt:
    packages:
    - clang-9
    - gcc-8
```
I never tried it personally but it seems relatively popular.